src/core/ApplicationInfo.cpp

```diff
@@ -94,7 +94,9 @@ ApplicationInfo::ApplicationInfo(QObject *parent): QObject(parent)
 #endif
     // Get fonts from rcc
     const QStringList fontFilters = {"*.otf", "*.ttf"};
+    const QStringList backgroundMusicFilters = {QString("*.%1").arg(COMPRESSED_AUDIO) };
     m_fontsFromRcc = QDir(":/gcompris/src/core/resource/fonts").entryList(fontFilters);
+    m_backgroundMusicFromRcc = QDir(":/gcompris/data/backgroundMusic").entryList(backgroundMusicFilters);
 
```

Comment:
This is too static. Either m_backgroundMusicFromRcc has to be updated whenever the resource is registered. Maybe even better: don't hold it in a member-variable just make ApplicationInfo::getBackgroundMusicFromRcc() return dynamically QDir(":/gcompris/data/backgroundMusic").entryList(backgroundMusicFilters); if it works